framework/src/packets/icmp/v6/ndp/router_advert.rs

```diff
@@ -318,7 +348,36 @@ mod tests {
 
             assert!(slla_found);
         } else {
-            panic!("bad packet");
+            panic!("no source link layer address option found");
+        }
+    }
+
+    #[dpdk_test]
+    fn find_mtu() {
```

Comment:
this feels like an unnecessary test. it reads like almost a duplicate of `test_add_source_link_layer_address`. you can modify that one to push multiple options and they are essentially the same thing.

Reply:
removed